gslib/commands/rpo.py

```diff
@@ -116,6 +121,48 @@ class RpoCommand(Command):
       },
   )
 
+  gcloud_storage_map = GcloudStorageMap(
+      gcloud_command={
+          'get':
+              GcloudStorageMap(
+                  gcloud_command=[
+                      'alpha', 'storage', 'buckets', 'list', _GCLOUD_LIST_FORMAT
+                  ],
+                  flag_map={},
+                  supports_output_translation=True,
+              ),
+          'set':
+              GcloudStorageMap(
+                  gcloud_command={
+                      'enforced':
```

Comment:
Did you try running the command with use_gcloud_storage=True in boto config? I think it should error out.

Reply:
Things like these would also be caught by adding a shim translation test like this: https://github.com/GoogleCloudPlatform/gsutil/blob/8cbf3362f34becf358f2fcd60dc3f20bf08a9a17/gslib/tests/test_ls.py#L194